src/account/update.rs

```diff
@@ -147,10 +148,17 @@ impl AccountHandle {
                         account_index,
                         WalletEvent::NewOutput(Box::new(NewOutputEvent {
                             output: OutputDataDto::from(&output_data),
-                            transaction: transaction
+                            transaction: transaction.as_ref().map(|tx| TransactionPayloadDto::from(&tx.payload)),
+                            transaction_inputs: transaction
                                 .as_ref()
-                                .map(|(tx, _inputs)| TransactionPayloadDto::from(tx)),
-                            transaction_inputs: transaction.as_ref().map(|(_tx, inputs)| inputs).cloned(),
+                                .map(|tx| {
+                                    tx.inputs
+                                        .clone()
```

Comment:
We're not using transaction anymore so this shouldn't be necessary as well ?

Reply:
We only have Some(&transaction), so we need a clone

Reply:
Ah yeah it comes from the get, my bad